### ISSUE TITLE
Text Preview Stopped After Upgrade

### ISSUE THREAD

**Issue by rudolfschmidt** [ranger/ranger#1641]
#### Runtime Environment

- Operating system and version: Linux rs.com 5.2.1-arch1-1-ARCH #1 SMP PREEMPT Sun Jul 14 14:52:52 UTC 2019 x86_64 GNU/Linux
- Terminal emulator and version: rxvt-unicode (urxvt) v9.22 - released: 2016-01-23
- Python version: Python 3.7.3
- Ranger version/commit: local/ranger 1.9.2-1
- Locale: LANG=en_US.UTF-8

#### Current Behavior
After upgrade text preview stopped. When I delete scope.sh from local config dir it works again, when I copy scope.sh from /usr/share/doc/ranger/config/scope.sh to my current config dir in .config/ranger it stopps again.

#### Expected Behavior
that I can copy scope.sh to local font dir, customize it like activate image and video preview and also still be able to see preview for texts.

#### Context

delete all config dir to figure out which file affects the text preview to stop. figured out its scope.sh

**Comment by toonn** [comment]
It works when you run `ranger --clean`?

**Comment by rudolfschmidt** [comment]
no

**Comment by toonn** [comment]
Any errors in the log `w`? And to be clear, we're talking about `.txt` files, right?

**Comment by rudolfschmidt** [comment]
When I press w in a ranger --clean env I get a task overview. No task in. And Yes, the preview of txt files does not work.

**Comment by toonn** [comment]
Sorry, `W`.

**Comment by rudolfschmidt** [comment]
rs@rs.com /home/rs/note20190727235037320677475.txt
Message Log:
11:28:20 INFO ranger version: ranger 1.9.2
11:28:20 INFO Python version: 3.7.3 (default, Jun 24 2019, 04:54:02) [GCC 9.1.0]
11:28:20 INFO Locale: en_US.UTF-8
11:28:20 INFO Process ID: 9400

**Comment by toonn** [comment]
Nope. Could you do a screencap of it working and not working? With asciinema for example.
